src/modules/facility/facility.service.ts

```diff
@@ -294,20 +301,34 @@ export class FacilityService {
         LenderTypeCode: defaultValues.lenderTypeCode,
       },
       BorrowerParty: {
-        PartyIdentifier: facilityToTranform.obligorPartyIdentifier,
+        PartyIdentifier: facilityToTransform.obligorPartyIdentifier,
       },
       ServicingUser: {
         UserAcbsIdentifier: defaultValues.servicingUser.userAcbsIdentifier,
         UserName: defaultValues.servicingUser.userName,
       },
       CompBalPctReserve: compBalPctReserve,
-      CompBalPctAmount: facilityToTranform.forecastPercentage,
+      CompBalPctAmount: facilityToTransform.forecastPercentage,
       RiskMitigation: {
         RiskMitigationCode: defaultValues.riskMitigationCode,
       },
     };
   }
 
+  private updateAcbsFacilityRequestFieldsForIssueAndAmendExpiryDate(facilityToTransform: BaseFacilityRequestItemWithFacilityIdentifier) {
+    const description = this.buildFacilityDescription(facilityToTransform.productTypeName, facilityToTransform.exposurePeriod);
```

Comment:
Can we just move this back to the base generator as its used in both create and update?